locale/fi/LC_MESSAGES/django.po

```diff
@@ -19,367 +19,599 @@ msgstr ""
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
 "X-Translated-Using: django-rosetta 0.7.6\n"
 
+#: audit_log/models.py:6
+#, fuzzy
```

Comment:
AFAIK, Django will by default ignore all fuzzy translations (unless specifically instructed otherwise). So it might be a good idea to go through them.